prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -158,16 +160,11 @@ Prebid Server reads the OpenRTB privacy fields:
 - user.ext.consent
 - device.lmt
 
-#### Other OpenRTB Fields
-
-Prebid Server doesn't do any special processing on any other fields, but passes them
-all to the bid and analytics adapters.
-
 ### OpenRTB Extensions
 
 #### Conventions
 
-OpenRTB 2.5 permits exchanges to define their own extensions to any object from the spec.
+OpenRTB 2.x permits exchanges to define their own extensions to any object from the spec.
```

Comment:
We don't support OpenRTB 2.6 yet, but starting to make more generic references for some sections where it makes sense now.